### ISSUE TITLE
Duplicate cursor, appear double mouse point when use rust ReskDesk

### ISSUE THREAD

**Issue by Pansvfj** [fufesou/RustDeskIddDriver#13]
我在被控端创建了RustDeskIddDriver的虚拟屏幕，当我使用RustDesk连接的时候出现了双鼠标。

https://github.com/user-attachments/assets/41934dd1-c986-4346-8bfd-16a3b61d864f

**Comment by fufesou** [comment]
Yes. It's a known issue. But I have no idea and no time to deal with this problem. 

> **H-Cursor** means hardware cursor support, without it you will get double cursor on some remote desktop apps. 

https://github.com/nomi-san/parsec-vdd
